coral/pkg/graphs/resource_models/Activity.json

```diff
@@ -354,6 +354,33 @@
                     "sortorder": null,
                     "visible": true
                 },
+                {
+                    "active": true,
+                    "cardid": "4954ccb4-0376-49d9-b5f6-77558752a88f",
+                    "component_id": "f05e4d3a-53c1-11e8-b0ea-784f435179ea",
+                    "config": null,
+                    "constraints": [],
+                    "cssclass": null,
+                    "description": null,
+                    "graph_id": "b9e0701e-5463-11e9-b5f5-000d3ab1e588",
+                    "helpenabled": false,
+                    "helptext": {
+                        "en": null
+                    },
+                    "helptitle": {
+                        "en": null
+                    },
+                    "instructions": {
+                        "en": null
+                    },
+                    "is_editable": false,
+                    "name": {
+                        "en": "National Grid References"
+                    },
+                    "nodegroup_id": "33b4430a-16be-11ef-8633-0242ac180006",
+                    "sortorder": null,
+                    "visible": true
+                },
```

Comment:
Activity also removes the experimental Easting and Northing group we had tried as well as adding the National Grid References nodegroup and the Irish Grid Node.
This is why it has much more in its diff